src/java.base/share/classes/java/net/JarURLConnection.java

```diff
@@ -172,14 +172,17 @@ private void parseSpecs(URL url) throws MalformedURLException {
             throw new MalformedURLException("no !/ found in url spec:" + spec);
         }
 
-        jarFileURL = new URL(spec.substring(0, separator++));
+        @SuppressWarnings("deprecation")
+        var tmp = jarFileURL = new URL(spec.substring(0, separator++));
+
```

Comment:
I realise that @SuppressWarnings needs a declaration here. I wonder if we could agree a better name for the unused variable name though, like `unusedSW` or something better?

Reply:
Having unnamed local variables[^1] would probably be best for this.

[^1]: https://openjdk.org/jeps/8294349

Reply:
How about `_unused` or `_unused1`, `_unused2` then in the meantime?

Reply:
I'd be happy to make the change. Let's wait to see if anybody has a better naming suggestion.

Reply:
Hello Daniel, I think calling it `unused` is fine. I did a quick search in the JDK code and we already have places where we use such a variable name for similar usecase. I don't see the need for using an underscore as a prefix, but it is OK with me if you and others prefer to use it.